### PR TITLE
[opentelemetry-collector] fix filesystem metrics for hostmetrics preset

### PR DESCRIPTION
**Pull request by povilasv**
Ref https://github.com/open-telemetry/opentelemetry-helm-charts/issues/537

Decided to go with the longer list from kube-prometheus-stack , those filesystem types most of them look like they are https://en.wikipedia.org/wiki/Synthetic_file_system

### CODE REVIEW

File: charts/opentelemetry-collector/pov-values.yaml [hunk 1]
```diff
@@ -0,0 +1,414 @@
+# Default values for opentelemetry-collector.
```
Comment (TylerHelmuth) [inline charts/opentelemetry-collector/pov-values.yaml:1]: Leftover from testing?

**Review by TylerHelmuth — APPROVED** [review]
Love it

**Review by Allex1 — APPROVED** [review]
pls rebase

**Comment by TylerHelmuth** [comment]
@povilasv please resolve conflicts

**Pull request merged**